Makefile

```diff
@@ -1,3 +1,11 @@
+TERRAFILE_VERSION=0.8
+ARM_TEMPLATE_TAG=1.1.6
+RG_TAGS={"Product" : "Get into teaching"}
```

Comment:
Should it be "Get into teaching website"?

Reply:
nice spot thanks  yes Updated!